Remove HTMLAttrxRef macro, part 1 (#25258)
mdn/content: files/en-us/web/http/redirections/index.md

{{HTTPSidebar}}

> **URL redirection**, also known as _URL forwarding_, is a technique to give more than one URL address to a page, a form, or a whole Website/application. HTTP has a special kind of response, called a **_HTTP redirect_**, for this operation.

Reviewer: ```suggestion
**URL redirection**, also known as _URL forwarding_, is a technique to give more than one URL address to a page, a form, a whole website, or a web application. HTTP has a special kind of response, called a **_HTTP redirect_**, for this operation.
```

These redirections are meant to last forever. They imply that the original URL should no longer be used, and replaced with the new one. Search engine robots, RSS readers, and other crawlers will update the original URL for the resource.

| Code  | Text                 | Method handling                                                                                         | Typical use case                                            |
| ----- | -------------------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |

> | `301` | `Moved Permanently`  | {{HTTPMethod("GET")}} methods unchanged. Others may or may not be changed to {{HTTPMethod("GET")}}. [1] | Reorganization of a Website.                                |
> | `308` | `Permanent Redirect` | Method and body not changed.                                                                            | Reorganization of a Website, with non-GET links/operations. |

Reviewer: ```suggestion
| `301` | `Moved Permanently`  | {{HTTPMethod("GET")}} methods unchanged. Others may or may not be changed to {{HTTPMethod("GET")}}. [1] | Reorganization of a website.                                |
| `308` | `Permanent Redirect` | Method and body not changed.                                                                            | Reorganization of a website, with non-GET links/operations. |
```

</head>
```

> The [`content`](/en-US/docs/Web/HTML/Global_attributes#content) attribute should start with a number indicating how many seconds the browser should wait before redirecting to the given URL. Always set it to `0` for accessibility compliance.

Reviewer: ```suggestion
The [`content`](/en-US/docs/Web/HTML/Element/meta#content) attribute should start with a number indicating how many seconds the browser should wait before redirecting to the given URL. Always set it to `0` for accessibility compliance.
```

### Keeping links alive

> When you restructure Websites, URLs change. Even if you update your site's links to match the new URLs, you have no control over the URLs used by external resources.

Reviewer: ```suggestion
When you restructure websites, URLs change. Even if you update your site's links to match the new URLs, you have no control over the URLs used by external resources.
```
